src/docs/product/data-management-settings/attachment-scrubbing.mdx

```diff
@@ -0,0 +1,130 @@
+
+- A minidump also contains a list of **code modules**,  which are loaded at the time the minidump is created.
+  This module list contains information about each module, including two fields that may contain file paths: the path of the code file and the path of the associated debug file.
+
+  Since these fields are pathnames, they have the potential to contain a user's home directory.
+  As a result, it makes sense to apply the *Usernames in filepaths* [data type](../advanced-datascrubbing/#data-types) to these fields.
+  The fields can be selected individually by their respective field names:
+
+  * `code_file`
+  * `debug_file`
+
+  Both fields have a [value type](../advanced-datascrubbing/#value-types) of `$string`. Any modification will exclude the basename of the pathnames to ensure the event processing pipeline can still operate correctly on the minidump.
+
+  This example is how the UI displays a rule to scrub these fields:
+
+  ```
+  [Remove] [Usernames in filepaths] from [$minidump.code_file || $minidump.debug_file]
+  ```
+
+  As this rule is more generic, it is also possible to apply it more generically using the [value type](../advanced-datascrubbing/#value-types):
+
+  ```
+  [Remove] [Usernames in filepaths] from [$string]
+  ```
+
+- Linux minidumps can contain both the **command line** and **environment variables**, which are included as copies of the files in `/proc/$pid/cmdline` and `/proc/$pid/environ`. Since these are direct copies of these files, they have this file format: NULL-separated records of `argv` arguments or `VAR=val` pairs respectively.
+
+  To select these, you can currently use only their [value types](../advanced-datascrubbing/#value-types): `$binary`, which will be successful because these are currently the only fields in the minidump with a binary value type.
+
+  This example is how the UI displays a rule to scrub the home directory from the `$HOME` env var:
+
+  ```
+  [Remove] [HOME=[^\u0000+]\u0000] from [$minidump.$binary]
+  ```
+
+  Note the syntax to denote a string delimited by NULL in the regular expression: any sequence of characters not containing NULL, but ending in NULL. Here, NULL itself is represented by its unicode codepoint: `\u0000`.
+
+
+## Malformed minidumps
+
+If a minidump fails to be parsed because it is malformed it could still contain PII.
+For this reason the minidump attachment is also scrubbed as one binary field if it can not be parsed.
+
+In this case the entire attachment can be selected as normal using a filename selector or a wildcard: `$attachments.'minidump.dmp'` or `$attachments.*`.
+Additionally since the entire attachment is now a single field it can be selected using the `$binary` [value type](../advanced-datascrubbing/#value-types).
+
+
+## Example
+
+A process usually contains a struct with it's environment variables early in the stack memory, in C semantics this can often this can be consumed by the process using the `char *envp[]` parameter ot `main()`.
```

Comment:
```suggestion
A process usually contains a struct with it's environment variables early in the stack memory, in C semantics this can often be consumed by the process using the `char *envp[]` parameter to `main()`.
```